samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -18,7 +18,8 @@
               "sqs:DeleteMessage",
               "sqs:DeleteMessageBatch",
               "sqs:GetQueueAttributes",
-              "sqs:ReceiveMessage"
+              "sqs:ReceiveMessage",
+	      "sqs:GetQueueUrl"
```

Comment:
```suggestion
              "sqs:GetQueueUrl"
```

Reply:
Should be resolved in commit 913e9ea7c131edff0dffae4576f50ba57b12a770